### ISSUE TITLE
Dependency lines for devel/subversion is missing the package name

### ISSUE THREAD

**Issue by dlangille** [FreshPorts/freshports#180]
The Dependency lines for [devel/subversion](https://www.freshports.org/devel/subversion/) is missing the package name:

```
* Dependency lines:
${PYTHON_PKGNAMEPREFIX}>0:devel/subversion
```

That should be:

```
* Dependency lines:
subversion>0:devel/subversion
```

Perhaps the clue is in:

`PKGNAME: subversion`

**Comment by dlangille** [comment]
See also https://www.freshports.org/editors/vim-console/

**Comment by dlangille** [comment]
See also #216

**Comment by sambler** [comment]
That would be a slave port failure. Looks like a slave port doesn't see the PORTNAME from its parent, or thinking it is a python module, looks at the wrong variable.

The parent port has `PYTHON_USES=python` for an option, that would be where the `${PYTHON_PKGNAMEPREFIX}` comes from, the slave port should remove that through OPTIONS_EXCLUDE but it ends up with `python:env` in USES.

**Comment by dlangille** [comment]
It's `PKGNAME` which come into play here:

```
[dan@devgit-ingress01:/var/db/freshports/ports-jail/var/db/repos/ports/editors/vim-console] $ make -V PORTNAME
vim
[dan@devgit-ingress01:/var/db/freshports/ports-jail/var/db/repos/ports/editors/vim-console] $ make -V PKGNAME
vim-console-8.2.2725
[dan@devgit-ingress01:/var/db/freshports/ports-jail/var/db/repos/ports/editors/vim-console] $ make -V USES
cpe iconv ncurses pkgconfig shebangfix python:env
[dan@devgit-ingress01:/var/db/freshports/ports-jail/var/db/repos/ports/editors/vim-console] $ 
```

The above examples are how `FreshPorts` pulls the information from the `Makefile`. It does this after first doing a `chroot` into the `/var/db/freshports/ports-jail` directory (hence the name `ports-jail`, but it's not really a jail) as the `freshports` user.

**Comment by dlangille** [comment]
The code in question:

https://github.com/FreshPorts/freshports/blob/0f850cdcff61ba865e8739f15df4a97bb5e18ac5/classes/port-display.php#L457

**Comment by dlangille** [comment]
`PKGNAME` is computed at run time and may not reflect what is actually installed. For example, the package name could by `py37-foo`, or `py36-foo` depending on what version of Python is used.

However, that does not apply to this situation. The code is wrong.

**Comment by dlangille** [comment]
This is fixed by 6cc3f803
